tools/github-release-asset.ts

```diff
@@ -0,0 +1,51 @@
+import { program } from 'commander'
+import fs from 'node:fs'
+
+import authenticate from './github-authentication.js'
+
+type Options = {
+  deliverinoPrivateKey: string
+  owner: string
+  repo: string
+  releaseId: number
+  files: string
+}
+
+const uploadAsset = async ({ deliverinoPrivateKey, owner, repo, releaseId, files }: Options) => {
```

Comment:
:upside_down_face: 
```suggestion
const uploadAssets = async ({ deliverinoPrivateKey, owner, repo, releaseId, files }: Options) => {
```
Also you are inconsistent between `asset` and `files`. Perhaps you could adjust that.

Reply:
I use both wordings, but they are slightly different:
files: These are the files on the disk (of circle ci)
assets: These are the assets of the github release.
It think it makes sense to keep both wordings as they are different. Think of it like an attachment of an email, where you attach a file.